services/121-service/seed-data/fsp/fsp-africas-talking.json

```diff
@@ -8,21 +8,19 @@
     {
```

Comment:
All of the changes within the `services/121-service/seed-data` folder were applied through a custom script (so no manual work was involved).

Reply:
Nice! There are also a lot of formatting changes in those json files. For me these are automatically edited on save if I open a vscode window from the `121-service` folder.

You can also run `npm run test:prettier -- --write` from the root folder. Maybe we should have something that `npm run fix` also runs that command

Reply:
Good that you mention it! These should be picked up automatically, and CI should complain about them if they aren't correct. I thought I address this in https://github.com/global-121/121-platform/pull/5191 but evidently something has slipped through the cracks. Will take a look and report back ✌️